lib/src/bin/phylum-cli.rs

```diff
@@ -225,11 +228,207 @@ fn handle_submission(api: &mut PhylumApi, config: Config, matches: clap::ArgMatc
     exit_status
 }
 
+/// Register a user. Drops the user into an interactive mode to get the user's
+/// details.
+fn handle_auth_register(api: &mut PhylumApi, config: &mut Config, config_path: &str) {
+    let name: String = Input::with_theme(&ColorfulTheme::default())
+        .with_prompt("Your name")
+        .interact_text()
```

Comment:
It looks like `interact_*` can fail for various i/o issues.  Instead of `unwrap`ing we should probably bubble up the error via `handle_auth` and then exit process with an error code if this is the case.